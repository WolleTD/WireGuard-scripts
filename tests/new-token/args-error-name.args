peer/1 172.16.1.1/32
