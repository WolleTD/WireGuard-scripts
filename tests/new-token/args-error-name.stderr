Error: Not a valid hostname!
